spec/05-classes-and-objects.md

```diff
@@ -502,11 +502,14 @@ definition apply to all constituent definitions.  The rules governing
 the validity and meaning of a modifier are as follows.
 
 ### `private`
-The `private` modifier can be used with any definition or
-declaration in a template.  Such members can be accessed only from
-within the directly enclosing template and its companion module or
+The `private` modifier can be used with any definition or declaration in a
+template. Private members of a template can be accessed only from within the
+directly enclosing template and its companion module or
 [companion class](#object-definitions).
 
+The `private` modifier is also valid for
+[top-level](09-top-level-definitions.html#packagings) templates.
```

Comment:
This doesn't say what the semantics are; I think you deleted it in an edit.

One might imagine a top-level private class C is private to object C instead of enclosing private[p].

Reply:
I suggested not to repeat the semantics here, as they are defined in the linked section.

Reply:
OK thanks for clarifying. I always forget to click the link. Please don't assume all your readers are so lazy.